ui/src/i18n/en.json

```diff
@@ -20,6 +20,7 @@
         "bitRate": "Bit rate",
         "discSubtitle": "Disc Subtitle",
         "starred": "Favourite",
+        "rating": "Rating",
```

Comment:
I think you are still missing the `rating` field for albums and artists in this file.